lib/client/index.js

```diff
@@ -87,7 +87,9 @@ client.init = function init(serverSettings, plugins) {
   client.language = language;
   
   client.hashauth = require('../hashauth');
-  client.hashauth.init(client, $).initAuthentication();
+  client.hashauth.init(client, $).initAuthentication(function afterAuth(isAuthenticated) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['isAuthenticated' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1563800288/issues/source?bid=2452379&fileBranchId=3019946#l90)